lib/pages/game_config/game_config.page.dart

```diff
@@ -190,15 +207,15 @@ class _GameConfig extends State<GameConfigPage> {
               //         ],
               //       )),
               // ),
+              ButtonWidget(
+                buttonType: ButtonType.primary,
+                contentType: ContentType.text,
+                function: () => () {},
+                title: AppLocalizations.of(context)!.start.toString(),
```

Comment:
Unless I am mistaken, You don't need toString at the end
```suggestion
                title: AppLocalizations.of(context)!.start,
```